model_analyzer/config/input/config.py

```diff
@@ -399,7 +398,7 @@ def objective_list_output_mapper(objectives):
                 'max_retries',
                 flags=['-r', '--max-retries'],
                 field_type=ConfigPrimitive(int),
-                default_value=100,
+                default_value=1000,
```

Comment:
It is not a good idea to have random "magic" numbers. A better way is to define a const somewhere. In C++, that may be in a top level `constants.h` file. But in python, there should be a simpilar way. One file that has all important constants. This both encourages reuse of the constant and provides just one place to make the configuration change.

Also, this would fit the pattern needed for the modes. A single parent class/interface that has default values `default.get_max_retries()`, then a different child class for offline and online modes. Then use that instantiated class for default values through out the project.